tests/testqualification/qualification/qual_workflows/2_metrics/kri0001b_custom.yaml

```diff
@@ -0,0 +1,62 @@
+meta:
+  Type: Metric
+  ID: kri0001b_custom
+  File: kri0001b_custom.yaml
+  GroupLevel: Race
+  Abbreviation: AE
+  Metric: Adverse Event Rate
+  Numerator: Adverse Events
+  Denominator: Days on Study
+  Model: Normal Approximation
+  Score: Adjusted Z-Score
+  AnalysisType: rate
+  Threshold: -5,-3,3,5
+  nMinDenominator: 300
+spec:
+  Analysis_Input:
+    subjid:
```

Comment:
I think `Analysis_Input` should have the columns of `gsm::analyticsInput`, which are SubjectID, GroupID, GroupLevel, Numerator, Denominator, and Metric.

Reply:
Added it in, swapped out Site for GroupLevel to prevent any further confusion as well